authors/apps/authentication/tests_/test_base_class.py

```diff
@@ -111,7 +111,8 @@ def setUp(self):
         self.article = {
             "title": "hello worlfd",
             "description": "desctriptuo",
-            "body": 'boddydydabagd'
+            "body": 'boddydydabagd',
+            "all_tags": ["andela", "xmen"]
```

Comment:
@kdaglas this has to be `tags` not `all_tags`.

Reply:
oh yes, let me change that